go/erasablekv/erasable_kv_store.go

```diff
@@ -43,20 +48,20 @@ type ErasableKVStore interface {
 type FileErasableKVStore struct {
 	libkb.Contextified
 	sync.Mutex
-	storagePath string
+	storageDir string
 }
 
 var _ ErasableKVStore = (*FileErasableKVStore)(nil)
 
 func NewFileErasableKVStore(g *libkb.GlobalContext) *FileErasableKVStore {
 	return &FileErasableKVStore{
 		Contextified: libkb.NewContextified(g),
-		storagePath:  g.Env.GetDataDir(),
+		storageDir:   getStorageDir(g),
 	}
 }
 
 func (s *FileErasableKVStore) filepath(key string) string {
-	return filepath.Join(s.storagePath, key)
+	return filepath.Join(s.storageDir, key)
```

Comment:
Preexisting q: Should we be escaping the key somehow? It seems like it would cause problems if a slash ended up in here. `encodeURIComponent` or similar maybe?

Reply:
yeah i think that would be best, i'll open a PR after this to address